### PR TITLE
Update coding_style.md

### PR DESCRIPTION
**Pull request by Piterden**

### CODE REVIEW

File: conventions/coding_style.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This style is used in the standard library. You can use it in your own project t
 
 ## Naming
 
-__Type names__ are camelcased. For example:
+__Type names__ are StudlyCased. For example:
```
Comment (RX14) [inline conventions/coding_style.md]: I've never heard anyone call it this?

Comment (straight-shoota) [reply conventions/coding_style.md]: Yeah, `CamelCased` please 👍

Comment (Glazy) [reply conventions/coding_style.md]: I've typically found people refer to this as [PascalCase](https://techterms.com/definition/pascalcase).

```suggestion
__Type names__ are PascalCased. For example:
```

**Review by straight-shoota — APPROVED** [review]

**Review by beta-ziliani — APPROVED** [review]
Thanks @Piterden !

**Pull request merged**